astroquery/cadc/core.py

```diff
@@ -0,0 +1,193 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""
+=============
+Cadc TAP plus
+=============
+
+"""
+
+from astroquery.cadc.tap import TapPlus
+
+__all__ = ['Cadc', 'CadcTAP']
+
+
+class CadcTAP(object):
```

Comment:
Probably, I did it this way because that's how gaia did it and now there are a couple of function that have code in them to get everything to work right